src/index.html

```diff
@@ -8,9 +8,47 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <form
+    class="page__search-bar search-bar search-bar--large"
+    action="#"
+    method="post"
+    data-qa="big"
+    >
+      <div class="search-bar__rectangle">
+        <div class="search-bar__search-icon"></div>
```

Comment:
if you add this icon as bg image to input, then no need to use this wrappers, just `<form><input/></form>`